src/Sulu/Bundle/MediaBundle/Tests/Functional/Controller/MediaControllerTest.php

```diff
@@ -256,6 +274,8 @@ public function testGetById()
      */
     public function testcGet()
```

Comment:
I guess it stands for collection get. Was not introduced by us, is a convention from the FOSRestBundle.